content/departments/engineering/dev/admin-exp/devops/managed/v1.1/mi1-1_creation_process.md

```diff
@@ -53,31 +54,33 @@ For basic operations like accessing an instance for these steps, see [managed in
    mg create-oidc-secret --client-id=<CLIENT_ID_FROM_OAUTH_CREDENTIALS> --client-secret=<CLIENT_SECRET_FROM_OAUTH_CREDENTIALS>
    ```
 
-1. Initialize the instance and copy the password reset link that is generated
+1. Initialize the instance and copy the password reset link that is generated by running
 
-Note: if added in MI template, add CE email in `config.yaml`:
+   > Note: if requested in the new MI request issue, add the CE email in `config.yaml`:
+   > ```
+   > additionalAdmins:
+   > - CE_NAME@sourcegraph.com
+   > ```
 
-```
-additionalAdmins:
-- CE_NAME@sourcegraph.com
-```
-
-```bash
-mg init-instance $CUSTOMER_ADMIN_EMAIL
-```
+   ```bash
+   mg init-instance $CUSTOMER_ADMIN_EMAIL
+   ```
 
 1. Go back to `terraform.tfvars` and set `enable_alerting` to `true`. Run `terraform apply` and verify that only `google_monitoring_alert_policy.primary` is created.
+1. Commit all changes
 1. Enable application performance metrics via
 
-```bash
-mg --customer=$COMPANY sync-alerts
-```
+   ```bash
+   mg sync-alerts
+   ```
+
+1. Enable metrics collection and GCP alerts for the new instance:
 
-1. Enable metrics collection and GCP alerts for the new instance by following [these instructions](https://github.com/sourcegraph/deploy-sourcegraph-managed/tree/main/monitoring#2-add-new-managed-instances-project-to-be-monitored)
```

Comment:
@filiphaftek I removed the link here because it is a bit confusing considering the steps are already described below